docs/graph_analytics_workloads.md

```diff
@@ -1,27 +1,43 @@
+## What is Graph Analytics
+Broadly speaking, any kind of computation over graph data can be regarded as graph analytics. We find that computation patterns of different graph analytics vary a lot: some only involve a small number of vertices/edges, while others access a large fraction of (even all) vertices/edges of a graph. In GraphScope, we call the former as *graph traversal*, while use the term *graph analytics* to refer to the latter, unless otherwise specified. 
 
-Currently, there exist diverse types of graph analytics algorithms, which usually iteratively access a large fraction of (even all) vertices/edges of a graph to explore underlying insights hidden in graph data. Typical graph analytics algorithms include general analytics algorithms (e.g., PageRank, shortest path, and maximum flow), community detection algorithms (e.g., maximum clique/bi-clique, connected components, Louvain and label propagation), graph mining algorithms (e.g., frequent structure mining and graph pattern discovery). 
+Currently, there exist diverse types of graph analytics algorithms, which usually iteratively access a large fraction of (even all) vertices/edges of a graph to explore underlying insights hidden in graph data. Typical graph analytics algorithms include general analytics algorithms (e.g., PageRank, the shortest path, and maximum flow), community detection algorithms (e.g., maximum clique/bi-clique, connected components, Louvain and label propagation), graph mining algorithms (e.g., frequent structure mining and graph pattern discovery). These graph analytics algorithms have been widely applied in real scenarios. For example,  as shown in the following figure, Google uses PageRank to rank web pages in their search engine results, and the shortest path algorithm can help path planning in logistics and delivery services.
 
-In GraphScope, Graph Analytics Engine (GAE) is responsible for handling such graph analytics algorithms, which allows users to easily define and implement their algorithms while providing efficient execution for these algorithms.
+![Applications of graph analytics](./images/analytics_examples.png)
 
-### Rich built-in graph analytics algorithms
 
-GAE of GraphScope provides 20 graph analytics algorithms as built-in algorithms, and users can directly invoke them. The build-in algorithms contain most commonly used algorithms, including PageRank, BFS, DFS, shortest path and LCC. In addition, GraphScope is compatible with NetworkX APIs, and thus diverse kinds of [built-in algorithms in NetworkX](https://networkx.org/documentation/stable/reference/algorithms/index.html) can also be directly invoked by users. In total, over 100 build-in graph analytical algorithms can be directly executed over GraphScope, without any developing effort.
+## Characteristics of Graph Analytics Workloads

```

Comment:
## Challenges of Graph Analytics on Large Graphs.
- for small graphs, well studied.